tilemap.c

```diff
@@ -29,7 +29,7 @@ tile_map_new(int size, size_t item_size, TileMapItemFreeFunc item_free_func)
     self->size = size;
     self->item_size = item_size;
     self->item_free_func = item_free_func;
-    const int map_size = 2*self->size*2*self->size;
+    const float map_size = 2*self->size*2*self->size;
```

Comment:
It looks like the size is affected by HEAVY_DEBUG in operationqueue.c.  I need to verify that I didn't turn debug and that's what fixed it.  I don't think I'm suddenly allocating 25GB of ram and not noticing .. .